schema/handler.go

```diff
@@ -108,3 +110,39 @@ func (h *Handler) get(w http.ResponseWriter, r *http.Request, ps httprouter.Para
 		return
 	}
 }
+
+// Raw JSON Schema IDs
+//
+// swagger:model jsonSchemaIDs
+// nolint:deadcode,unused
+type jsonSchemaIDs []string
+
+// nolint:deadcode,unused
+// swagger:parameters getJsonSchemaIDs
+type getJsonSchemaIDs struct{}
+
+// swagger:route GET /schemas v0alpha1 getJsonSchemaIDs
+//
+// Get all JSON Schema IDs
+//
+//     Produces:
+//     - application/json
+//
+//     Schemes: http, https
+//
+//     Responses:
+//       200: jsonSchemaIDs
+//       500: jsonError
+func (h *Handler) getIDs(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	ids, err := h.r.IdentityTraitsSchemas(r.Context()).GetIDs()
```

Comment:
I think it would make more sense to just return the complete schemas instead, and add pagination?

Reply:
Yes, but that approach is much more effort and I don't know enough about the architecture of kratos to do it. I also just need a solution that works and not something fancy

Reply:
Ok, I understand - but Ory Kratos is used by thousands and we need solutions that work in general and not just for individuals. As it is right now, I feat that we can not accept it. Adding the required changes however shouldn't be too hard. We have helpers for pagination:

https://github.com/ory/kratos/blob/c2c5a588a4f0e6a1876ee7cfec772e2499120ebe/identity/handler.go#L111-L112

If you don't want to work on the suggestions I fear we need to close this PR

Reply:
I can try doing it. It will just take me a bit longer than someone who knows kratos better.